LiveSplit.AutoSplitters.xml

```diff
@@ -23692,4 +23692,15 @@
         <Type>Script</Type>
         <Description>Autosplitting available (by Biksel)</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>S.T.A.L.K.E.R. 2: Heart of Chornobyl</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/KusochekShaurmi/HeartOfChornobyl_LoadRemover/refs/heads/main/loadremoverHeartOfChornobyl.asl</URL>
+            <URL>https://github.com/just-ero/asl-help/raw/7cd7f36aab142860c0a6a9f9a855bb44d7264b04/lib/asl-help</URL>
```

Comment:
You don't use asl-help in your script. Why do you need it?

Reply:
Just didnt really understand what asl help do in code. Copy that from another autosplitter.